### DISCUSSION TITLE
OpenWRT GLInet Router installation

### DISCUSSION THREAD

**Discussion by C0nw0nk** [Barre/privaxy#37] [Q&A]
So I have the following flash router https://www.gl-inet.com/products/gl-ax1800/#specs

The good thing about privaxy is it allows to run URL filters instead of DNS filters as with the YouTube APP on phones for example this is a brilliant soloution.

But not sure if your project is compatible with OpenWRT flash routers.

Any update on when you can also secure your admin login page ?

The router does already use AdGuard home what is only blocking DNS level ad domains but the features of Ublock Origin URL filtering lists and javascript capabilities makes it so much better and more powerful.

**Comment by adriangalera** [comment]
I have the same router and unfortunately the compiled binaries do not work, I guess it's an incompatible arm arch:
```bash
root@GL-AX1800:~# uname -a
Linux GL-AX1800 4.4.60 #1 SMP PREEMPT Fri Feb 11 12:37:59 CST 2022 armv7l GNU/Linux
```
